crypto/keys/pkcs11/no_pkcs11.go

```diff
@@ -0,0 +1,29 @@
+// +build !pkcs11
```

Comment:
Do we need this build flag at all? Maybe make it unconditional?

What is the cost? Extra package import?

Reply:
There's *two* `pkcs11.FromProto` defined, one in `pkcs11.go`, the other in `no_pkcs11.go`, and they should never "meet". 😃